### ISSUE TITLE
Length doesn't work on wildcards

### ISSUE THREAD

**Issue by Rhynorater** [OJ/gobuster#159]
When you have wildcards enabled gobuster does not show the content length anymore. This was my best way of telling whether or not the wildcard had changed (simply letting it run and `grep -v`ing for the normal content size. This would give me anomalous results. 

PoC:
```
gobuster dir -w ./wordlist.txt -u poc.rhynorater.com/test.php -l --wildcard
```

**Comment by FireFart** [comment]
Hi,
which version of gobuster are you running? We had some errors in the 3.0.0 beta which are fixed in 3.0.1. I tried your command with the latest version and the version is output.

**Comment by FireFart** [comment]
Closing this for now. If there are still errors with the latest version feel free to reopen this issue
